### PR TITLE
make fewer calls to marshalTags

### PR DESCRIPTION
**Pull request by dgnorton**

### CODE REVIEW

**Comment by toddboom** [comment]
:+1:
And :100: more for awesome branch naming.

File: commands.go [hunk 1]
```diff
@@ -125,9 +125,10 @@ type dropMeasurementCommand struct {
 }
 
 type createMeasurementSubcommand struct {
-	Name   string              `json:"name"`
-	Tags   []map[string]string `json:"tags"`
-	Fields []*Field            `json:"fields"`
+	Name          string              `json:"name"`
+	Tags          []map[string]string `json:"tags"`
+	MarshaledTags []string            `json:"marshaledTags"`
```
Comment (otoolep) [inline commands.go]: No need to export this, so perhaps make it `marshaledTags` so it's clear to the next guy.

Comment (otoolep) [inline commands.go]: A comment would also be nice.

**Comment by otoolep** [comment]
+1 once we don't export it.

**Pull request merged**